.travis.yml

```diff
@@ -60,6 +60,7 @@ install:
   # installing regex on jython fails (jython can't compile the native extension), this awk filter prevents it
   - cat requirements*.txt | awk \"${JYTHON:-False}'" == "True" && /^regex/ || ! /^regex/' | xargs pip install
   - if [ -z "$SCA" ]; then python setup.py develop; else echo skip; fi
+  - git clone https://github.com/9livesdata/whylog.git; cd whylog; python setup.py develop; pip install -r requirements.txt; cd -
```

Comment:
according to http://python-packaging.readthedocs.io/en/latest/dependencies.html, `python setup.py develop` already takes care of dependencies, so `pip install -r requirements.txt` will do nothing.